lib/linux_kernel/processor.c

```diff
@@ -36,24 +36,22 @@
 #include "lib/trace.h"
 
 /**
-   @addtogroup processor
-
-   @section proc-kernel Kernel implementation
+   Convert bitmap from one format to another. Copy cpumask bitmap to m0_bitmap.
 
-   This file includes additional data structures and functions for processing
-   processors data - for kernel-mode programs.
+   @param dest -> Processors bitmap for Motr programs.
+   @param src -> Processors bitmap used by Linux kernel.
+   @param bmpsz -> Size of cpumask bitmap (src)
 
-   This file will also implement Linux kernel-mode processors interfaces.
+   @pre Assumes memory is alloacted for outbmp and it's initialized.
 
    @see lib/processor.h
-
-   @{
+   @see lib/bitmap.h
  */
 
-#ifndef CONFIG_X86_64
-#error "Only X86_64 platform is supported"
-#endif
-
+//#ifndef CONFIG_X86_64
+//#error "Only X86_64 platform is supported"
+//#endif
```

Comment:
Remove these as we will have conditional compilation.

Reply:
removed